components/bpmn-q/modeler-component/extensions/planqk/PlanQKReplaceMenuProvider.js

```diff
@@ -37,15 +37,16 @@ export default class PlanQKMenuProvider {
     getPopupMenuEntries(element) {
         const self = this;
         return function (entries) {
-
+            
             // do not show entries for extension elements of other plugins
             if (!(element.type.startsWith('bpmn') || element.type.startsWith('planqk'))) {
                 return entries;
             }
 
             // add replacement entries for the active service subscription as replacements for a PlanQK service task
             if (is(element, consts.PLANQK_SERVICE_TASK)) {
-                return self.createPlanQKServiceTaskEntries(element, self.activeSubscriptions);
+                let serviceTaskEntries = self.createTaskEntries(element, self.activeSubscriptions);
+                return Object.assign(serviceTaskEntries, entries);
```

Comment:
This can completely override the entries in `serviceTaskEntries`. (`Object.assign(target, source)` assigns all values from source to the target)
But I guess, that this order is required to have the PlanQK entry at the top of the menu.

Reply:
Normally the keys of the entries are totally different from the servicetaskentries. In case we have the same key we can fix this by `Object.assign({}, serviceTaskEntries, entries);`.  So should I use the more "safe" option in case we have same keys?